pyproject.toml

```diff
@@ -68,7 +68,7 @@ zfpy = [
     "numpy<2.0.0",
 ]
 pcodec = [
-    "pcodec>=0.2,<0.3",
+    "pcodec>=0.3",
```

Comment:
Would this cause a problem for zarr users if Pcodec rolled out 0.4 or 1.0 and broke some part of the API used here? I don't think that's likely anytime soon, but why remove the upper bound?

Reply:
It would, and that's what happened with 0.3. Definitely safer, and fine with me.

On the other hand, `numcodecs` doesn't seem to have any sort of upstream test job, so it will be up to us to manually identify breaking vs non-breaking changes and manually bump the version constraint. If you thought the likelihood of any more breaking API changes like this (for `standalone` and `ChunkConfig`) was unlikely until 1.0, then we could constrain only to major version?

Reply:
I think breaking changes are unlikely to happen soon, but if/when they eventually do, they're likely to affect numcodecs, regardless of 0.4 or 1.0. So I'd suggest we keep the upper bound. Ultimately up to numcodecs maintainers though.